src/lib/i18n/locales/languages.json

```diff
@@ -23,6 +23,14 @@
 		"code": "ca-ES",
 		"title": "Catalan"
 	},
+        {
+                "code": "ceb-PH",
+                "title": "Cebuano (Filipino)"
+        },
+	{
+		"code": "ceb-PH",
```

Comment:
Duplicate

Reply:
Removed duplicate entry, sorry and thank you!